components/Image/Image.tsx

```diff
@@ -11,6 +12,17 @@ import NextImage, { ImageLoader } from 'next/image';
 
 type Props = ComponentProps<typeof NextImage>;
 
+type Type = {
+	alt: string,
+	src: string,
+	key?: string,
+	srcset?: string,
+	width?: number,
+	originalWidth?: number,
+	height?: number,
+	originalHeight?: number,
+} & Props;
```

Comment:
a few things here:
- let's try to keep prop types named as `Props` for consistency
- some of these properties are redundant if you are joining with `NextImage` prop type
- `key` is a built-in prop for all React components so you never need to specify it

```suggestion
type Type = {
	src: string,
	srcset?: string,
	originalWidth?: number,
	originalHeight?: number,
} & ComponentProps<typeof NextImage>;
```